lib/screens/ranks/rank_screen.dart

```diff
@@ -13,43 +13,376 @@ class _RankScreen extends State<RankScreen> {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: AppBar(backgroundColor: Colors.blueAccent, actions: [
-        GestureDetector(
-          onTap: () {
-            Get.snackbar(
-              '새로운 쓰레기 획득!',
-              '도감에 새로운 쓰레기가 등록되었어요!\n확인해볼까요?',
-              icon: const Icon(
-                Icons.check_circle_sharp,
-                size: 30,
-                color: Colors.white,
-              ),
-              onTap: (snackbar) {
-                Get.to(() => const SettingScreen());
-              },
-              overlayColor: ColorSystem.primary,
-              duration: const Duration(seconds: 3),
-              mainButton: TextButton(
-                onPressed: () {},
-                child: const Text(
-                  '확인하기',
-                  style: TextStyle(color: Colors.white),
+      // appBar: AppBar(backgroundColor: Colors.blueAccent, actions: [
+      //   GestureDetector(
+      //     onTap: () {
+      //       Get.snackbar(
+      //         '새로운 쓰레기 획득!',
+      //         '도감에 새로운 쓰레기가 등록되었어요!\n확인해볼까요?',
+      //         icon: const Icon(
+      //           Icons.check_circle_sharp,
+      //           size: 30,
+      //           color: Colors.white,
+      //         ),
+      //         onTap: (snackbar) {
+      //           Get.to(() => const SettingScreen());
+      //         },
+      //         overlayColor: ColorSystem.primary,
+      //         duration: const Duration(seconds: 3),
+      //         mainButton: TextButton(
+      //           onPressed: () {},
+      //           child: const Text(
+      //             '확인하기',
+      //             style: TextStyle(color: Colors.white),
+      //           ),
+      //         ),
+      //         colorText: Colors.white,
+      //         backgroundColor: ColorSystem.primary.withOpacity(0.8),
+      //         barBlur: 0.5,
+      //       );
+      //     },
+      //     child: const Icon(Icons.star, size: 50),
+      //   ),
+      // ]),
+      body: Container(
+        decoration: const BoxDecoration(
+          gradient: LinearGradient(
+              begin: FractionalOffset.topCenter,
+              end: FractionalOffset.bottomCenter,
+              colors: [
+                Color.fromRGBO(125, 178, 98, 1),
```

Comment:
```suggestion
                ColorSystem.primary
```
저희 앱에서 사용하는 초록색이면 이거 쓰면 되여 맨 위에 styles import해주시구요